test/commands/break_test.rb

```diff
@@ -108,6 +108,46 @@ def test_setting_breakpoint_to_invalid_location_shows_an_error
     end
   end
 
+  #
+  # Tests adding breakpoints to empty methods
+  #
+  class BreakAtEmptyMethodsTest < TestCase
+    def program
+      strip_line_numbers <<-RUBY
+          1:  module Byebug
+          2:    #
+          3:    # Toy class to test breakpoints
+          4:    #
+          5:    class #{example_class}
+          6:      def self.a
+          7:      end
+          8:
+          9:      def b
+         10:
```

Comment:
Is the extra empty line intentional?

Reply:
Yes it is intentional to test lineno 10 is not displayed.

Reply:
:+1: